obstacleAvoidanceWIthSmartCar/src/obstacleAvoidanceWithSmartCar/obstacleAvoidanceWithSmartCar.ino

```diff
@@ -19,12 +19,21 @@ BrushedMotor rightMotor(12, 13, 11);
 DifferentialControl control(leftMotor, rightMotor);
 SmartCar car(control, gyroscope, leftOdometer, rightOdometer);
 
+//declaring pins for ultrasonic sensors
+const int TRIGGER_PIN_REAR = A4;
+const int ECHO_PIN_REAR = A5;
 //variables for the obstacle avoidance
 const int TRIGGER_PIN = 6; //D6
 const int ECHO_PIN = 5; //D5
+
 const unsigned int MAX_DISTANCE = 30;
+
+SR04 back (TRIGGER_PIN_REAR, ECHO_PIN_REAR, MAX_DISTANCE);
 SR04 front(TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);
-int distance;
+int distanceFront;
+int distanceBack;
```

Comment:
I would strongly suggest that you initialize these to `0` or something. They will not necessarily get the value you expect them to have.

Reply:
Absolutely, thanks !

Reply:
I've got a question: 
there is an option to set "assignee" when you create the pull request.
 OR you can go into the issue and assign people in there. which one is it that makes life easier for all of us ? Cheers

Reply:
@Fredrik93 I think assigning yourselves to issues is more clear since assigning a pull request might imply that the assignee is doing the code review.